vuln/npm/1.json

```diff
@@ -15,7 +15,7 @@
   ],
   "vulnerable_versions": "<=1.5.1",
   "patched_versions": ">=1.5.2",
-  "overview": "A vulnerability exists in bassmaster <= 1.5.1 that allows for an attacker to provide arbitrary JavaScript that is then executed server side via eval.",
+  "overview": "\"A vulnerability exists in bassmaster <= 1.5.1 that allows for an attacker to provide arbitrary JavaScript that is then executed server side via eval.\"",
```

Comment:
Why adding a `"` around them?

Reply:
I'll remove the unnecessary things and make the changes.